development/installed-packages.json

```diff
@@ -59,7 +86,7 @@
             "direct-dependency": true,
             "homepage": "http://simplesamlphp.org",
             "source": "https://github.com/simplesamlphp/simplesamlphp",
-            "version": "v1.18.9",
+            "version": "v1.18.8",
```

Comment:
Comment: It's always strange when things go back a version.

Reply:
Agreed